src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -535,7 +535,7 @@ private static void CheckBaseAddress(Uri baseAddress, string parameterName)
 
         private void SetTimeout(CancellationTokenSource cancellationTokenSource)
         {
-            Contract.Requires(cancellationTokenSource != null);
+            Debug.Assert(cancellationTokenSource != null);
```

Comment:
Can we remove the using imports for System.Diagnostics.Contracts in some of these files where Contract.Assert was the only use of contracts?